wsimod/nodes/storage.py

```diff
@@ -490,16 +543,6 @@ def __init__(
         )
         self.muptNpar = 0.001  # [kg/m2/day] nitrogen macrophyte uptake rate
         self.muptPpar = 0.0001  # 0.01, # [kg/m2/day] phosphorus macrophyte uptake rate
```

Comment:
I can see that these aren't used - but I guess they were used in CatchWat? Are we likely to need them again? If so, maybe just copy this excerpt into #2

Reply:
probably not as they are for the solids and we haven't decided which sedimentation-resuspension model we are going to integrate ... plus these parameters are not performing very well in previous case studies